tree/readspeed/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+# Copyright (C) 1995-2022, Rene Brun and Fons Rademakers.
+# All rights reserved.
+#
+# For the licensing terms see $ROOTSYS/LICENSE.
+# For the list of contributors see $ROOTSYS/README/CREDITS.
+
+############################################################################
+# CMakeLists.txt file for building ROOT readspeed package
+# @author Bertrand Bellenot CERN
+############################################################################
+
+ROOT_OBJECT_LIBRARY(ReadSpeed
+  src/ReadSpeed.cxx
+  src/ReadSpeedCLI.cxx
+)
+
+add_dependencies(ReadSpeed RIO Tree TreePlayer)
```

Comment:
@bellenot why `add_dependencies` rather than the target-oriented `target_link_libraries`?

Reply:
Not sure if we really link against those. I'll check. As I said, that was a quick solution to help going further. I'll spend more time on this

Reply:
@dkpoult we had a brief discussion with Bertrand and we think this line can be removed, can you try it out please?